### PR TITLE
themes: add Material Ocean to v2

### PR DESCRIPTION
**Pull request by Blacksuan19**
Signed-off-by: blacksuan19 <i@blacksuan19.me>

### CODE REVIEW

**Comment by morpheusthewhite** [comment]
What are the differences in the UI from the default look?

**Comment by Blacksuan19** [comment]
just the colors for now

File: THEMES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Themes preview
 
 Here you can find a preview of all the themes. Some of them may have different colour schemes (in that case you'll find different entries in the same theme, each one named after the colour scheme).
+
+## Material Ocean
+
+![Psycho Screenshot](Material-Ocean/screenshot.png)
```
Comment (Milo123459) [inline THEMES.md:7]: Should say Material Ocean Screenshot

Comment (Milo123459) [reply THEMES.md:7]:
```suggestion
![Material Ocean Screenshot](Material-Ocean/screenshot.png)
```

**Comment by morpheusthewhite** [comment]
> just the colors for now, you can see a screenshot in the repo https://github.com/material-ocean/Spotify-Theme/

In that case, I would prefer not to create a separate theme for that: I just created a Default theme in `v2` and pushed your color scheme

**Pull request closed without merging**